config.json

```diff
@@ -59,6 +59,15 @@
       ]
     },
     {
+      "uuid": "6eba4eac-8395-4ad6-ac21-3651a11ab4b4",
+      "slug": "collatz-conjecture",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
```

Comment:
since this exercise now uses Maybe, please add it to the topics list (see "rna-transcription" for example of how it is done)

Reply:
ok